spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -104,7 +104,7 @@
 
       get :timeout
 
-      expect(flash[:timeout]).to eq t('session_timedout')
+      expect(flash[:timeout]).to eq t('session_timedout', app: 'login.gov', minutes: 8)
```

Comment:
use `APP_NAME` instead of `login.gov` ? Also, can we ref `Figaro.env.session_timeout_in_minutes` so "8" is not a magic number?